### DISCUSSION TITLE
Skipped deployments of monorepo projects create noise in Github PR timeline

### DISCUSSION THREAD

**Discussion by michalsanger** [vercel/vercel#5492] [General]
Would it be possible not to publish messages from the Vercel bot about inactive deployments of monorepo projects into PR timeline on Github?

If the build of a monorepo project was skipped due to "Ignored build step", there is still a message in the PR timeline from Vercel bot about it with label "Inactive". This creates a visual noise and gets only worse when there is more projects in the monorepo or some changes were pushed.

**Comment by michaelbirchler** [comment]
A screenshot of the visual noise: 
<img width="481" alt="Screenshot 2021-01-13 at 10 19 29" src="https://user-images.githubusercontent.com/24622/104432084-ee7cc680-5588-11eb-92f0-20847fb76f52.png">

I think a good idea could be to list the deployments with a preview URL prominent and all canceled deployments as a small plaintext list or grouped by deployment status.

**Comment by michaelbirchler** [comment]
This screenshot also documents the Vercel bot noise very well. If the pull request has a lot of commits, the Vercel bot comments hundreds of status messages to the whole pull request and all the relevant comments are really hard to find.
<img width="941" alt="image" src="https://user-images.githubusercontent.com/24622/113606992-02e0d480-9649-11eb-9d1d-81a55b27d8bf.png">

**Reply by michaelbirchler** [reply]
I found a temporary solution to mute all vercel comments with the `silent` property inside `vercel.json`
```json
{
    "github": {
        "silent": true
    }
}
```
https://vercel.com/support/articles/how-to-prevent-vercel-github-comments
